lib/kasket.rb

```diff
@@ -39,4 +39,17 @@ def self.cache_store=(options)
   def self.cache
     @cache_store ||= Rails.cache
   end
+
+  def self.pending_records
+    Thread.current['kasket_pending_records']
```

Comment:
I had it that way, then I changed it to be consistent with the other uses of `Thread.current`
https://github.com/zendesk/kasket/blob/master/lib/kasket/configuration_mixin.rb#L9-L17

I could change them all to use symbols...